### DISCUSSION TITLE
Cannot start the frontend page according to the tutorial, error 418 is prompted.

### DISCUSSION THREAD

**Discussion by changeyoucant** [remix-run/remix#9279] [Q&A]
![image](https://github.com/remix-run/remix/assets/51197525/d0480fed-8f42-4ad0-b45c-6130f99f0470)
![image](https://github.com/remix-run/remix/assets/51197525/1169d7bd-8d93-4bcb-97d7-104930534ffe)

**Comment by nexneo** [comment]
We are experiencing exactly same issue in production build. @changeyoucant did you find any root cause? 

For us same code that was working last week is failing, no changes in code, build process or infrastructure.

**Comment by nexneo** [comment]
It got fixed now, I think. https://github.com/remix-run/remix/issues/9275
